Just use the 1, otherwise we would have "too many arguments to function `iaxc_initialize'"
--- simpleclient/tkphone/iaxcli.c.orig	Mon Apr  7 18:05:28 2008
+++ simpleclient/tkphone/iaxcli.c	Wed Aug  7 20:30:01 2013
@@ -413,7 +413,7 @@ int main(int argc, char **argv) {
 #endif
 
     atexit(iaxc_shutdown); /* activate the exit handler */
-    if (iaxc_initialize(AUDIO_INTERNAL_PA,1)) {
+    if (iaxc_initialize(1)) {
 	fatal_error("cannot initialize iaxclient!");
     }
 
